react-select/react-select.d.ts

```diff
@@ -156,6 +156,17 @@ declare namespace ReactSelect {
         onFocus?: __React.FocusEventHandler;
         onInputChange?: (inputValue: string) => void;
         onOptionLabelClick?: (value: string, event: Event) => void;
+
+        /**
+         * function invoked when the menu opens
```

Comment:
actually i notice now that the other event handler callbacks above (`onFocus`, etc) don't have doc comments because i figured they were obvious. so let's either remove these comments or add similarly obvious ones to the other callbacks. my preference is to just remove.